src/parser/type_checking.c

```diff
@@ -4930,6 +4930,31 @@ pt_get_expression_definition (const PT_OP_TYPE op, EXPRESSION_DEFINITION * def)
       sig.return_type.val.type = PT_TYPE_JSON;
       def->overloads[num++] = sig;
 
+      def->overloads_count = num;
+      break;
+    case PT_JSON_PRETTY:
+      num = 0;
+
+      /* two overloads */
+
+      /* arg1 */
+      sig.arg1_type.is_generic = false;
+      sig.arg1_type.val.type = PT_TYPE_JSON;
+
+      /* return type */
+      sig.return_type.is_generic = false;
+      sig.return_type.val.type = PT_TYPE_CHAR;
+      def->overloads[num++] = sig;
+
+      /* arg1 */
```

Comment:
I don't think we need this overload.
Fetch function expects JSON type and this leads to CHAR not JSON.